modules/core/src/main/scala/gem/Step.scala

```diff
@@ -9,6 +9,37 @@ sealed abstract class Step[A] extends Product with Serializable {
   def instrument: A
 }
 
+object Step {
+  final case class Location(loc: List[Int]) {
+    override def toString: String =
+      loc.mkString("{", ",", "}")
+  }
```

Comment:
We may end up using the same structure elsewhere (and it will certainly get built out more) so it might be useful to make a top-level data type.